.circleci/config.yml

```diff
@@ -69,34 +87,30 @@ jobs:
       - store_test_results:
           path: reports/junit
 
-  test-node-8:
+  test-node-11:
     working_directory: ~/jest
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:11
     steps:
       - checkout
       - restore-cache: *restore-cache
       - run: yarn --no-progress
       - save-cache: *save-cache
       - run:
-          command: yarn test-ci-partial
+          command: yarn test-ci
       - store_test_results:
           path: reports/junit
 
-  test-node-6:
+  test-browser:
     working_directory: ~/jest
     docker:
-      - image: circleci/node:6
+      - image: circleci/node:8-browsers
```

Comment:
does `10-browsers` exist?

Reply:
Yep. If we're gonna update that, can we update `test-or-deploy-website` to `node:10` too or is there a reason why it's 8?

Reply:
For sure, LTS in all the places